tests/DCPS/Compiler/maps/test.idl

```diff
@@ -0,0 +1,34 @@
+#if defined __TAO_IDL_FEATURES
+#  include __TAO_IDL_FEATURES
+#  if TAO_IDL_HAS_MAP
+    struct TestStruct {
+        int32 id;
+        string msg;
+    };
+
+    enum TEST_ENUM {TEST1, TEST2, TEST3};
+
+    typedef sequence<TestStruct> TestStructSeq;
+    typedef map<int32, TestStruct> TestStructMap;
+    // typedef map<int32, string> TestIntStringMap; //TODO Not compiling
```

Comment:
Does this need to resolved before the PR is merged?

Reply:
The TODO may have been an artifact from something else works fine now